aluuu/osm_xml
# Upgrade dependencies

This PR upgrades dependencies for osm_xml. Previously, the library requires core version < v0.11, and now it's updated with latest release (v.0.14.1). Upgraded dependencies for ounit as well for unit tests to work. (fix #1 )

## How to test
`make && make test` (might need to run after `ocaml setup.ml -configure --enable-tests`)

Expected to see: 
```
Ran: 6 tests in: 0.04 seconds.
OK
```
